src/distPlus/distribution/AlgebraicShapeCombination.re

```diff
@@ -115,18 +114,23 @@ let toDiscretePointMassesFromTriangulars =
 };
 
 let combineShapesContinuousContinuous =
```

Comment:
If this does the combining through convolution, I think that should be part of the function name, as we also combine with sampling.

Reply:
I agree. I'll have to think of a more accurate name though, convolution doesn't really describe the process well :thinking:

Reply:
Maybe we could refer to these as ``Numerical`` solutions or similar, which is different from ``Symbolic`` or ``Sampling``(not sure about the last one).

If we change the file name, that should make it obvious, as all the functions in it use Numerical instead of Sampling techniques.